beancount_import/source/schwab_csv.py

```diff
@@ -59,7 +60,7 @@
 
 * Not all Schwab "actions" (transaction types) are supported. There's no reference for all
 the possible actions, and Schwab could add new ones any time. If your CSV includes an
-unsupported action, you'll get a `ValueError: 'Foo' is not a valid SchwabAction`. Please
+unsupported action, you'll get a `ValueError: 'Foo' is not a valid BrokerageAction`. Please
```

Comment:
Could you also add a note here about the possibility of `ValueError: 'Foo' is not a valid BankingEntryType`? Unless you're confident that Schwab will never add any banking entry types you haven't already seen.